src/psyclone/tests/parse/algorithm_test.py

```diff
@@ -1,7 +1,7 @@
 # -----------------------------------------------------------------------------
 # BSD 3-Clause License
 #
-# Copyright (c) 2019, Science and Technology Facilities Council.
+# Copyright (c) 2019-2021, Science and Technology Facilities Council.
```

Comment:
There are various pylint warnings in this file (reimport of ParserFactory, imports from fparser are not grouped, lots of useless suppression of unused-argument).

Reply:
Fixed complaints.